libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -74,4 +81,49 @@ export class StripeManager {
       customer.tax?.automatic_tax === 'not_collecting'
     );
   }
+
+  /**
+   * Updates customer object with incoming tax ID if existing tax ID does not match
+   *
+   * @param customerId Customer ID of customer to be updated
+   * @param taxId Customer tax ID to be updated if different from existing tax ID
+   * @returns Updated customer, or undefined if existing customer tax ID matches
+   */
+  async setCustomerTaxId(customerId: string, taxId: string) {
+    const customerTaxId = await this.getCustomerTaxId(
+      customerId,
+      MOZILLA_TAX_ID
+    );
+
+    if (customerTaxId !== taxId || customerTaxId === '') {
+      return await this.client.updateCustomer(customerId, {
+        invoice_settings: {
+          custom_fields: [{ name: MOZILLA_TAX_ID, value: taxId }],
+        },
+      });
+    }
+
+    return;
+  }
+
+  /**
+   * Returns tax ID of customer
+   *
+   * @param customerId ID of customer to fetch and check existing tax ID
+   * @param taxName Custom field name for given tax ID
+   * @returns The tax ID of customer or empty string if not found
```

Comment:
I think a more JS-like return value here would be a `string | undefined`, where string is returned if a value was found, and `undefined` is returned if not found.

Reply:
Sounds good. Updated!